<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapshotDialog</class>
 <widget class="QDialog" name="SnapshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Take a Screenshot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="takeScreenshotLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Take Screenshot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="gtRadio">
       <property name="text">
        <string>Grab groundtruth screen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sampleRadio">
       <property name="text">
        <string>Grab test sample screen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="spesificAreaRadio">
       <property name="text">
        <string>Select area to grab </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,10,0">
      <item row="0" column="0">
       <widget class="QLabel" name="targetFolderLabel">
        <property name="text">
         <string>Target Folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="folders">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>/home</string>
        </property>
        <property name="maxCount">
         <number>10</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>/home</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="browseButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="CancelOkButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CancelOkButtons</sender>
   <signal>accepted()</signal>
   <receiver>SnapshotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelOkButtons</sender>
   <signal>rejected()</signal>
   <receiver>SnapshotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
